images/win/Windows2016-Readme.md

```diff
@@ -13,6 +13,7 @@
 - Go 1.15.15
 - Julia 1.6.2
 - Kotlin 1.5.20
+- LLVM 12.0.1
```

Comment:
@henti , We shouldn't update autogenerated doc files. Please remove all changes.

Reply:
> > @henti sorry for such a late response, but it seems we forget to mention that pester tests and software readme should also be updated. I suggest adding tests for all the main tools included in the llvm choco bundle (version output should be enough). And the same tools should be mentioned in the software report, somewhere in Language and Runtime section probably

What **software readme** is being referred to then ? 
Do I need to generate the updates ? 

Some documentation on what is required to add additional dependencies to the repo would be very helpful.

Reply:
You should update - https://github.com/actions/virtual-environments/blob/23a859a9bccbd4dff15840d4cacf9f04f31181fe/images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1#L33

Reply:
I've added as requested and removed the edits to the autogenerated files.